src/goto-instrument/cover_instrument.h

```diff
@@ -44,7 +44,14 @@ class cover_instrumenter_baset
   {
     Forall_goto_program_instructions(i_it, goto_program)
     {
-      instrument(goto_program, i_it, basic_blocks);
+      // do not instrument if instruction has a source location but is marked as
+      // originating from a java virtual function call removal
+      if(
```

Comment:
This is misplaced - it should be implemented as a goal filter, see e.g. https://github.com/diffblue/cbmc/blob/develop/src/goto-instrument/cover_filter.cpp#L96

Reply:
done, it's now part of `bool internal_goals_filtert::operator()(const source_locationt &source_location) const`